service/controller/kvm/v16/version_bundle.go

```diff
@@ -8,9 +8,9 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "TODO",
-				Description: "TODO",
-				Kind:        versionbundle.KindChanged,
+				Component:   "cluster-operator",
+				Description: "Added support for creating a cluster configmap.",
```

Comment:
```suggestion
				Description: "Added support for creating a cluster configmap.",
```

```suggestion
				Description: "Added support for creating a cluster configmap for use by managed apps.",
```

I think we should mention what the configmap is for. @puja108 WDYT?

Also we need the changelog for all 3 controllers.

Reply:
I always missed it. Thanks for the notice.